tethys_portal/templates/header.html

```diff
@@ -5,7 +5,7 @@
   <nav id="header-navbar" class="navbar navbar-expand-md navbar-dark shadow px-4 py-3 mx-3" style="background-color: {{ site_globals.primary_color|default:'#0a62a9' }};">
     <div class="container-fluid">
       {% block header_brand %}
-        <a id="header-brand" class="navbar-brand" href="{% url 'home' %}">
+        <a id="header-brand" class="navbar-brand" href="{% if single_app_mode %} {% if configured_single_app %} {% url configured_single_app.index_url %}  {% else %} {% url 'user:profile' %} {% endif %} {% else %} {% url 'home' %} {% endif %}">
```

Comment:
Change this to use the App Name and App icon and don't show the app library link in single app mode.